services/app-api/src/postgres/contractAndRates/updateDraftRate.ts

```diff
@@ -1,24 +1,20 @@
+import { findRateWithHistory } from './findRateWithHistory'
+import type { NotFoundError } from '../storeError'
 import type {
-    ActuaryCommunication,
-    ActuaryContact,
-    PrismaClient,
     RateCapitationType,
-    RateDocument,
-    RateSupportingDocument,
-    RateType as PrismaRateType,
-} from '@prisma/client'
-import { findRateWithHistory } from './findRateWithHistory'
+    RateType as DomainRateType,
+    ActuaryCommunicationType,
+    ActuaryContact,
+    SubmissionDocument,
+} from 'app-web/src/common-code/healthPlanFormDataType'
```

Comment:
Yay, thank you, I missed this. My rule for non-domain types is that they should never escape the package they are relevant to. So since the editable types are arguments that are going to be passed in from our resolvers, they should be based on domain types so that nothing outside of the postgres package ever has to import something from prisma

Reply:
Yep! Makes sense. This was an error from my code editor autocomplete I think - because the two types have almost the exact same names I didn't notice the discrepancy until I started writing the field by field tests.